components/bl-lottieAnimation-component/README.md

```diff
@@ -0,0 +1,43 @@
+# Lottie Animation
+
+Lottie Animation is a component of Backendless UI-Builder designer. This allows you to upload and add animation in your page.
+
+The component based on external [Lottie Animation](https://github.com/airbnb/lottie-web).
+## Usage
+
+### Styles and Settings
+
+<dl>
+<dt>Data</dt>
+<dd>Allows write json for animation.</dd>
+<dt>Type</dt>
+<dd>Allows select type of animation (svg, canvas, Html). Default value 'svg'.</dd>
+<dt>Loop</dt>
+<dd>Allows to specify looping or non-looping animation. Default value 'true'</dd>
+<dt>Stopped</dt>
+<dd>Allows to specify stopped or not stopped animation.</dd>
+</dl>
+
+### Event Handlers and Bindable Properties
+
+<dl>
+<dt>Data Logic</dt>
+<dd>Allows to specify data for animation in Backendless logic</dd>
```

Comment:
Could we add a little instruction with links where user could find/create an animation?

Reply:
@StasivPavlo ?